x/gov/client/cli/query_test.go

```diff
@@ -381,3 +383,53 @@ func (s *CLITestSuite) TestCmdQueryVote() {
 		})
 	}
 }
+
+func (s *CLITestSuite) TestCmdGeConstitution() {
+	testCases := []struct {
+		name         string
+		args         []string
+		expCmdOutput string
+	}{
+		{
+			"get constitution with json response",
+			[]string{
+				"1",
+				fmt.Sprintf("--%s=json", flags.FlagOutput),
+			},
+			"1 --output=json",
```

Comment:
There is no argument output supported by the command constitution, so ti should actually fail

Reply:
This test isnt supposed to be in there actually. Its a copy of the other but wrong lol

Reply:
Okay, maybe it could be nice to add that output flag and display it as json too. I believe most commands of gov allow that, so imo this one too.